.github/workflows/ci.yml

```diff
@@ -49,8 +50,18 @@ jobs:
              -Plicense-check \
             | tee ${{ env.MVN_BUILD_OUTPUT_FILE }}
 
+          mvn ${MVN_COMMON_OPTIONS} dependency:tree -Plicense-check | tee ${{ env.MVN_DEPENDENCY_PLUGIN_OUTPUT_FILE }}
+
       - name: Check licensing
         run: |
-          mvn ${MVN_COMMON_OPTIONS} exec:java@check-licensing -N \
+          mvn ${MVN_COMMON_OPTIONS} exec:java@check-license -N \
```

Comment:
nit: can we move it into its own step rather than having it included in the Build step? ...to have a clearer separation and easier check what went wrong if something went wrong

Reply:
IIUC it is already in a separate step...
Or do you mean a separate job?

Reply:
moved `mvn dependency:tree` to check shade optional step